### PR TITLE
Pmm 9452 change admin user password3

### PR DESCRIPTION
**Pull request by rasika-chivate**
Change the password for the default admin user.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -152,3 +152,20 @@ docker exec -t pmm-server bash -c  "grafana-cli --homepath /usr/share/grafana a
 ```
 
 (This example assumes your Docker container is named `pmm-server`.)
+
+
+## How to change the PMM password for a deafult admin user?
```
Comment (nikita-b) [inline docs/faq.md:157]:
```suggestion
## How to change the PMM password for a default admin user?
```
deafult ->  default

File: docs/faq.md [hunk 2]
```diff
@@ -152,3 +152,20 @@ docker exec -t pmm-server bash -c  "grafana-cli --homepath /usr/share/grafana a
...
+
+    ```sh
+    docker exec -t pmm-server change-admin-password <new_password>
+    ```
+
+    * For PMM versions prior to 2.27.0:
+
+        ```sh
+        docker exec -t pmm-server bash -c 'grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password newpass'
+        ```
```
Comment (nikita-b) [inline docs/faq.md:171]: Why do we have other indents here?

File: docs/setting-up/server/docker.md [hunk 3]
```diff
@@ -56,7 +56,22 @@ How to run PMM Server with Docker based on our [Docker image].
...
+
+    ```sh
+    docker exec -t pmm-server change-admin-password <new_password>
+    ```
+
+    * For PMM versions prior to 2.27.0:
+
+        ```sh
+        docker exec -t pmm-server bash -c 'grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password newpass'
+        ```
```
Comment (nikita-b) [inline docs/setting-up/server/docker.md:71]: The same question about indents.

File: docs/faq.md [hunk 4]
```diff
@@ -152,3 +152,20 @@ docker exec -t pmm-server bash -c  "grafana-cli --homepath /usr/share/grafana a
 ```
 
 (This example assumes your Docker container is named `pmm-server`.)
+
+
+## How to change the PMM password for a deafult admin user?
+
+If you're deploying through Docker you can use change the password for an admin user after starting the Docker container as follows:
```
Comment (nikita-b) [inline docs/faq.md:159]: Maybe will be better:
```suggestion
If you're deploying through Docker you need to change the default password for an admin user after starting the Docker container as follows:
```

**Review by nikita-b — CHANGES_REQUESTED** [review]

**Review by nikita-b — APPROVED** [review]

**Review by denisok — APPROVED** [review]

**Pull request merged**